.ci/cico-plugin-registry-pr-test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# shellcheck disable=SC1091,SC1090
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+set -e
+set -x
+
+#Download and import the "common-qe" functions
+export IS_TESTS_FAILED="false"
+DOWNLOADER_URL=https://raw.githubusercontent.com/eclipse/che/master/tests/.infra/centos-ci/common-qe/downloader.sh
+curl $DOWNLOADER_URL -o downloader.sh
+chmod u+x downloader.sh
+. ./downloader.sh
+
+setup_environment
+setConfigProperty "test.workspace.devfile.url" "https://raw.githubusercontent.com/che-incubator/che-java-tests/master/devfile.yaml"
+
+export TAG="PR-${ghprbPullId:?}"
+export IMAGE_NAME="quay.io/eclipse/che-plugin-registry:$TAG"
+CHE_SERVER_PATCH="$(cat <<EOL
+spec:
+  server:
+    pluginRegistryImage: $IMAGE_NAME
+    selfSignedCert: true
+  auth:
+    updateAdminPassword: false
+    openShiftoAuth: false
+EOL
+)"
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+. "${WORK_DIR}"/../cico_functions.sh
+build_and_push
+
+installChectl
+yum install --assumeyes -d1 \
+    patch \
+    pcp \
+    wget \
+    bzip2 \
+    python3 \
+    firefox
+  python3 -m pip install selenium
+  wget https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-linux64.tar.gz
+  tar -xvzf geckodriver*
+  chmod +x geckodriver
+  mv geckodriver /usr/local/bin/
+
+startCheServer "$CHE_SERVER_PATCH"
+
+createTestWorkspace > workspace_url.txt
+workspace_url=$(tail -n 1 workspace_url.txt)
+echo "$workspace_url"
+
+oc project che
+oc get namespace
+oc get routes
+
+pods=$(oc get pods --all-namespaces -l che.workspace_id --field-selector status.phase=Running 2>&1)
+while [ "$pods" == 'No resources found.'  ];
+do
+    echo "No pod found with che.workspace_id"
+    echo "Current available pods are"
+    oc get pods
+    echo "Current deployments are"
+    oc get deployments
+    oc get pods --all-namespaces -l che.workspace_id
+    sleep 20
```

Comment:
should we have a shorted wait period ?

Reply:
It takes more than 1 minute until workspace's pod gets status Running, so, I don't think it'll change something if we decrease this value. Do you think to set timeout with 10 seconds will be better?

Reply:
yes, it's to avoid to spend time waiting (even if overall running job time is 15mn)